library/src/test/java/com/chuckerteam/chucker/internal/support/FormatUtilsTest.kt

```diff
@@ -58,4 +88,79 @@ class FormatUtilsTest {
             """.trimIndent()
         )
     }
+
+    @Test
+    fun testFormatHeaders_withNullValues() {
+        val result = FormatUtils.formatHeaders(null, false)
+        assertThat(result).isEmpty()
+    }
+
+    @Test
+    fun testFormatHeaders_withEmptyValues() {
+        val result = FormatUtils.formatHeaders(listOf(), false)
+        assertThat(result).isEmpty()
+    }
+
+    @Test
+    fun testFormatHeaders_withoutMarkup() {
+        val result = FormatUtils.formatHeaders(exampleHeadersList, false)
+        val expected = "Accept: text/html\nAuthorization: exampleToken\n"
+        assertThat(result).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatHeaders_withMarkup() {
+        val result = FormatUtils.formatHeaders(exampleHeadersList, true)
+        val expected = "<b> Accept: </b>text/html <br /><b> Authorization: </b>exampleToken <br />"
+        assertThat(result).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatByteCount_zeroBytes() {
+        val resultNonSi = FormatUtils.formatByteCount(0, false)
+        val resultSi = FormatUtils.formatByteCount(0, true)
+        val expected = "0 B"
+        assertThat(resultNonSi).isEqualTo(expected)
+        assertThat(resultSi).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatByteCount_oneKiloByte() {
+        val oneKb = 1024L
+        val resultNonSi = FormatUtils.formatByteCount(oneKb, false)
+        val resultSi = FormatUtils.formatByteCount(oneKb, true)
+        val expectedNonSi = "1.0 KiB"
+        val expectedSi = "1.0 kB"
+        assertThat(resultNonSi).isEqualTo(expectedNonSi)
+        assertThat(resultSi).isEqualTo(expectedSi)
+    }
+
+    @Test
+    fun formatXml_emptyString() {
+        assertThat(FormatUtils.formatXml("")).isEmpty()
+    }
+
+    @Test
+    fun formatXml_properXml() {
+        val xml =
+            """<example>value</example>"""
+        val expected = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n<example>value</example>\n"
```

Comment:
```suggestion
        val xml = """
                <example>value</example>
                """.trimIndent()
        val expected = """
                <?xml version="1.0" encoding="UTF-8" standalone="no"?>
                <example>value</example>
                
                """.trimIndent()
```

Probably easier to read